app/javascript/file-upload.js

```diff
@@ -0,0 +1,11 @@
+import MOJFrontend from '@ministryofjustice/frontend'
+import $ from 'jquery'
+window.$ = $
+
+if (typeof MOJFrontend.MultiFileUpload !== "undefined") {
+    new MOJFrontend.MultiFileUpload({
+        container: $(".moj-multi-file-upload"),
```

Comment:
If we are using jquery only for the element selection, can we try not to use jQuery, we can try rewrite this as : `document.querySelectorAll('.moj-multi-file-upload')`